tests/extmod/machine_uart_tx.py

```diff
@@ -14,6 +14,7 @@
     uart_id = 0
     tx_pin = "GPIO0"
     rx_pin = "GPIO1"
+    timing_margin_us = 160
```

Comment:
Looking at the achieved values of delta_us, it's 135 - 145µs for RP2 at 2400 Baud and it seems well distributed (without making real statistical tests). So 160µs does not look too tight.

![grafik](https://github.com/user-attachments/assets/99285bd3-73db-41fc-b25e-d2726930663e)

Reply:
Thanks for the thorough testing!